clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/task/TaskTracker.java

```diff
@@ -0,0 +1,13 @@
+package com.linkedin.venice.hadoop.task;
+
+/**
+ * An interface to report task health and track task progress for different execution engines
+ */
+public interface TaskTracker {
+  default void heartbeat() {
+  }
+
+  default float getProgress() {
+    return -1;
```

Comment:
Hmm why progress is a float? Could you add JavaDoc to the interface methods so it could be easier for reader?

Reply:
Oh yeah, I missed that. Will do it